<launch> 
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find piko_nav)/config/rtabmap_rgbd.rviz" output="screen"/>

  <!-- Construct and voxelize the point cloud (for fast visualization in rviz) -->
  <node pkg="nodelet" type="nodelet" name="points_xyzrgb" args="load rtabmap/point_cloud_xyzrgb standalone_nodelet">
    <remap from="rgb/image" to="/camera/rgb/image_rect_color"/>
    <remap from="depth/image" to="/camera/depth_registered/image_raw"/>
    <remap from="rgb/camera_info" to="/camera/depth_registered/camera_info"/>
    <remap from="cloud"           to="voxel_cloud" />

    <param name="rgb/image_transport" type="string" value="compressed"/>
    <param name="depth/image_transport" type="string" value="compressedDepth"/>
    
    <param name="queue_size" type="int" value="10"/>
    <param name="voxel_size" type="double" value="0.01"/>
  </node>
</launch>

